pkg/logql/bench/cmd/bench/main.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/charmbracelet/bubbles/list"
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/grafana/loki/v3/pkg/logql/bench"
+	"github.com/grafana/loki/v3/pkg/logql/bench/cmd/bench/views"
+)
+
+// mainModel represents the overall application state
+type mainModel struct {
+	currentView views.ViewID
+	listView    *views.ListView
+	runView     *views.RunView
+	width       int
+	height      int
+}
+
+func initialModel() mainModel {
+	return mainModel{
+		currentView: views.ListID,
+		listView:    views.NewListView(loadBenchmarks()),
+		runView: views.NewRunView(views.RunConfig{
+			Count:        1,
+			TraceEnabled: false,
+			Selected:     []string{},
+			StorageType:  "both", // Default to running both storage types
+		}),
+	}
+}
+
+func (m mainModel) Init() tea.Cmd {
```

Comment:
Is this fn part of an interface? If not, you could remove it

Reply:
required but charm yes.